.github/workflows/benchmark.yaml

```diff
@@ -98,7 +103,9 @@ jobs:
         run: |
           cd benchmarks/terraform/
 
+          # safety timeout for scraping
+          sleep 30
```

Comment:
Is there a way to somehow check that the metrics have been reported, instead of this sleep?

Reply:
Synced f2f, removed.